server/utils/placementRequests/adminIdentityBar.ts

```diff
@@ -23,18 +28,16 @@ export const adminActions = (placementRequest: PlacementRequestDetail): Array<Id
         text: 'Cancel placement',
       },
     ]
-  } else {
-    items = [
-      {
-        href: '#',
-        text: 'Create placement',
-      },
-    ]
   }
-
   return [
     {
-      items,
+      href: '#',
+      text: 'Create placement',
     },
   ]
 }
+
+export const title = (placementRequest: PlacementRequestDetail) => `
+<span class="govuk-caption-l">Placement request</span>
+<h1 class="govuk-heading-l">${placementRequest.person.name}</h1>
```

Comment:
It would be good to preempt the changes to the Person object here and add a case for where a Person doesn't have a name.